app/js/controller/list_controller.js

```diff
@@ -26,6 +26,14 @@ export default class ListController extends Controller {
     this.refreshInstalledList();
   }
 
+  showAlertDialog(msg) {
+    if (!this.alertDialog) {
+      this.alertDialog = document.querySelector('#alert-dialog');
+    }
```

Comment:
Can't put it in the ctor, because the alert-dialog hasn't been rendered at this point yet. I could put it in render with something like:
```
render() {
  super();
  this.alertDialog = document.querySelector('#alert-dialog');
}
```